src/redux/books/booksSlices.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable no-param-reassign */
```

Comment:
- [ ] It is unprofessional to disable eslint so I kindly suggest you fix the eslint issues rather than disabling them, they are there to assist you write better code that can be understood by anyone who checks and uses your code 👍